Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM alpine:edge
 MAINTAINER Konstantin Baierer <konstantin.baierer@gmail.com>
 
 EXPOSE 8080
-ADD . /ocr-fileformat
+COPY . /ocr-fileformat
```

Comment:
Looks good. I suggest to create a separate PR for this change, so it can be merged.